.gitlab.mk

```diff
@@ -113,7 +113,8 @@ package: deploy_prepare
 	if [ "$$(echo $(GC64) | sed 's/.*=//')" = ON ]; then                                                           \
 		export PRODUCT_NAME=tarantool-gc64; \
 		if [ "${OS}" = "ubuntu" ] || [ "${OS}" = "debian" ]; then \
-		    $$(sed -i'' -e 's/Package: tarantool$$/Package: tarantool-gc64/' debian/control); \
+			$$(mv debian debian_); \
```

Comment:
Looks like we need to add some cleanup step when all goes right and something goes wrong. I can propose such an approach:

```
_package: deploy_prepare
        <...>

package:
        export CLEANUP_CMD="mv debian debian-gc64 && mv debian_ debian"; \
        bash -c "trap 'trap - EXIT; ! [ -d debian_ ] || $${CLEANUP_CMD}' EXIT; $(MAKE) -f .gitlab.mk _package"
```

Reply:
I would propose to simplify the command to:
```Makefile
package:
	export CLEANUP_CMD="mv debian debian-gc64 && mv debian_ debian"; \
	trap 'if [ -d "debian_/" ]; then $$($CLEANUP_CMD); fi;' EXIT; \
```

Reply:
I've just tested this approach and it worked. If you don't insist, I would leave the logic in one target. Looks better for me.

Reply:
Okay, you are right. Fixed almost like you've proposed. Stoned bash syntax in Makefile...
```Makefile
package:
	echo "Creating the package"; \
	export CLEANUP_CMD="mv ./debian ./debian-gc64 && mv ./debian_ ./debian"; \
	bash -c "trap 'trap - EXIT; ! [ -d debian_ ] || ($${CLEANUP_CMD})' EXIT; $(MAKE) -f .gitlab.mk _package"

```